library/agent/Agent.ts

```diff
@@ -438,10 +430,30 @@ export class Agent {
       });
   }
 
-  onFailedToWrapMethod(module: string, name: string) {
+  onFailedToWrapMethod(module: string, name: string | undefined) {
     this.logger.log(`Failed to wrap method ${name} in module ${module}`);
   }
 
+  onFailedToWrapModule(module: string, error: Error) {
```

Comment:
Why does `onFailedToWrapMethod` accept `string | undefined` if we have this one?

Reply:
Because if a default export (root, not esm .default) is wrapped, there is no methodName. Can change to print default export instead.